tasks/translation/translate_content.py

```
@@ -0,0 +1,216 @@
+                    )
+                    out = resp.choices[0].message.content
+                    # naive parse
+                    lang = "unknown"
+                    conf = 0.0
+                    for part in out.splitlines():
+                        if part.lower().startswith("lang:"):
+                            lang = part.split(":", 1)[1].strip()
+                        if part.lower().startswith("confidence:"):
+                            try:
+                                conf = float(part.split(":", 1)[1].strip())
+                            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)